calendarsubscription/calendarsubscription.go

```diff
@@ -87,7 +92,12 @@ func (cs CalendarSubscription) Normalize() (*CalendarSubscription, error) {
 }
 
 func (cs CalendarSubscription) renderICalFromShifts(shifts []oncall.Shift, generatedAt time.Time) ([]byte, error) {
-	data := iCalRenderData{Shifts: shifts, ReminderMinutes: cs.Config.ReminderMinutes, Version: version.GitVersion(), GeneratedAt: generatedAt}
+	var uids []string
+	for i := 0; i < len(shifts); i++ {
+		sum := sha256.Sum256([]byte(cs.UserID + cs.ScheduleID + shifts[i].Start.String() + shifts[i].End.String()))
```

Comment:
Also, we should use the `UserID` from the shift instead of the subscription owner.

At the moment, they will (or should) be the same, but in the future, that will change (e.g. show all users on the calendar) and is not guaranteed.